tests/MyGame/Example/MonsterStorage_grpc.go

```diff
@@ -4,19 +4,16 @@
 
```

Comment:
What meaning it? I'm not use gofmt for this file.
That's just re-generate use fixed flatc, I fixed reault